wormchain/build/config/genesis.json

```diff
@@ -58,7 +58,11 @@
           "coins": [
             {
               "denom": "utest",
-              "amount": "10000"
+              "amount": "100000000"
+            },
+            {
+              "denom": "uworm",
+              "amount": "1000000000"
```

Comment:
Is this going to conflict with the worm tokens that are already minted elsewhere?

Reply:
It might, but this isn't the same as the mainnet genesis file which has removed uworm.  I don't see an issue with leaving it in devnet or just removing it as well.